test/__snapshots__/mathml-spec.js.snap

```diff
@@ -414,32 +423,34 @@ exports[`A MathML builder should render boldsymbol with the correct mathvariants
 <math>
   <semantics>
     <mrow>
-      <mrow>
-        <mi mathvariant="bold-italic">
-          A
-        </mi>
-        <mi mathvariant="bold-italic">
-          x
-        </mi>
-        <mn mathvariant="bold-italic">
-          2
-        </mn>
-        <mi mathvariant="bold-italic">
-          k
-        </mi>
-        <mi mathvariant="bold-italic">
-          ω
-        </mi>
-        <mi mathvariant="bold-italic">
-          Ω
-        </mi>
-        <mi mathvariant="bold-italic">
-          ı
-        </mi>
-        <mo mathvariant="bold-italic">
-          +
-        </mo>
-      </mrow>
+      <mi>
+        <mrow>
```

Comment:
This change doesn't seem right.

Reply:
I agree. This one may take some work. I'll get back to you tomorrow.

Reply:
@kevinbarabash Having looked closer at this, I now think it is correct. `\boldsymbol` makes a call to `\binrel` which return a mathord for argument that is not a single character. So to see this group wrapped by a `<mi>` is what we should expect.